resources/js/components/card/Avatar.vue

```diff
@@ -0,0 +1,31 @@
+<template>
```

Comment:
This component is special for the RandomGenerator and should therefore be in `resources/js/components/random/generator/user/AvatarCard.vue`, which makes the name `RandomGeneratorUserAvatarCard`.